src/main/java/io/pinecone/configs/PineconeConfig.java

```diff
@@ -50,17 +56,37 @@ public interface CustomChannelBuilder {
         ManagedChannel buildChannel();
     }
 
-    void validate() {
-        if (apiKey == null)
-            throw new PineconeValidationException("Invalid PineconeConfig: missing apiKey");
+    public void validate() {
+        if (apiKey == null || apiKey.isEmpty())
```

Comment:
Oo adding this 2nd check to my example app's code, thnx :)

Reply:
Oh java 🤦‍♂️